tap_surveymonkey/__init__.py

```diff
@@ -1,41 +1,30 @@
 #!/usr/bin/env python3
-import json
 import singer
-from tap_surveymonkey.data import SurveyMonkey
-from tap_surveymonkey.mode import sync, discover
 
+from .discover import discover
+from .sync import sync
```

Comment:
Please change the relative import statements to absolute imports as it more preferred way.